src/lib/validateToken.ts

```
@@ -1,40 +1,34 @@
-    log('No token found');
+    console.log('No token found');
     return false;
   }
 
   try {
     const res = await fetch('https://id.twitch.tv/oauth2/validate', {
-      headers: {
-        Authorization: `OAuth ${token}`,
-      },
+      headers: { Authorization: `OAuth ${token}` },
     }).then((notJson) => notJson.json());
```

ESLint

[error] fetch is not supported in op_mini all, IE Mobile 10, IE 10, bb 7

(compat/compat)